### PR TITLE
Expose TouchpadMagnify event

### PR DESCRIPTION
**Pull request by andi23rosca**
First PR contributing here :D

Noticed floem is not exposing the winit TouchpadMagnify event to the users

### CODE REVIEW

File: src/app_handle.rs [hunk 1]
```diff
@@ -234,7 +234,9 @@ impl ApplicationHandle {
             WindowEvent::MouseInput { state, button, .. } => {
                 window_handle.mouse_input(button, state);
             }
-            WindowEvent::TouchpadMagnify { .. } => {}
+            WindowEvent::TouchpadMagnify { delta, .. } => {
```
Comment (jrmoulton) [inline src/app_handle.rs:237]: This should also expose the phase

**Review by jrmoulton — CHANGES_REQUESTED** [review]
This looks good but I think it would be good to expose the phase as well

File: src/app_handle.rs (hunk 1 above)
Comment (andi23rosca) [reply src/app_handle.rs]: done ✅

**Comment by jrmoulton** [comment]
Thanks!

**Pull request merged**